mlflow/projects/databricks.py

```diff
@@ -167,7 +168,8 @@ def _run_shell_command_job(self, project_uri, command, env_vars, cluster_spec):
             # NB: We use <= on the version specifier to allow running projects on pre-release
             # versions, where we will select the most up-to-date mlflow version available.
             # Also note, that we escape this so '<' is not treated as a shell pipe.
-            "libraries": [{"pypi": {"package": "'mlflow<=%s'" % VERSION}}],
+            "libraries": [{"pypi": {"package": "'mlflow<=%s'" % VERSION}},
+                          *extra_libraries],
```

Comment:
Looks like this syntax is actually invalid in Python 2.7 FYI: https://travis-ci.org/mlflow/mlflow/jobs/592815196. That was the cause of some of the failures.

Reply:
MLflow is dropping support of Python 2 from 1.8 
https://mlflow.org/news/2020/03/08/python-2-end-of-support/index.html